### PR TITLE
update to libddwaf 1.0.16

### PR DESCRIPTION
**Pull request by vdeturckheim**
### Checklist

- [ ] The CHANGELOG.md has been updated
- [ ] Unit tests have been updated and pass
- [ ] If known, an appropriate milestone has been selected

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -180,7 +180,6 @@ Napi::Value DDWAFContext::run(const Napi::CallbackInfo& info) {
 
   DDWAF_RET_CODE code = ddwaf_run(this->_context, &data,
                                   &result, (uint64_t) timeout);
-  mlog("Run result action %i", result.action);
```
Comment (Anilm3) [inline src/main.cpp:183]: You can still log this, just use `code` instead as it contains the same value.

File: index.d.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@
 type rules = object;
 
 type result = {
+  timeout: boolean;
   perfData?: string;
   perfTotalRuntime?: number;
   data?: string;
```
Comment (simon-id) [inline index.d.ts:11]: `action` has been removed no ?

File: src/main.cpp (hunk 1 above)
Comment (vdeturckheim) [reply src/main.cpp:183]: good point

File: index.d.ts (hunk 2 above)
Comment (vdeturckheim) [reply index.d.ts:11]: no, it is still populated based on the return code :)

**Pull request merged**